### ISSUE TITLE
Image kubectl not found because of Kubernetes version

### ISSUE THREAD

**Issue by galexrt** [appscode/stash#266]
The operator fails to start when the Kubernetes version looks like this:
```
Server Version: version.Info{Major:"1", Minor:"8+", GitVersion:"v1.8.5+coreos.0", GitCommit:"b8e596026feda7b97f4337b115d1a9a250afa8ac", GitTreeState:"clean", BuildDate:"2017-12-12T11:01:08Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```
This fails because of the `+` in the minor version part.
Code ref: https://github.com/appscode/stash/blob/aa91dadd1ac97792a19e6d5470a6f80acdbb9e20/pkg/cmds/run.go#L61

Operator logs:
```
W0103 08:45:04.001676       1 client_config.go:529] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
F0103 08:45:05.073765       1 run.go:63] Image appscode/kubectl:1.8+.0 not found.
```

***

A fix would be to remove every non-numeric char from the minor version string.
What do you think should be changed here so it is fixed?

**Comment by tamalsaha** [comment]
https://github.com/appscode/stash/commit/ac4e8c93a982eb58b085a459779dd37edd986c01

**Comment by tamalsaha** [comment]
Just noticed that. in GKE:

```
 $ kubectl version 
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.3", GitCommit:"f0efb3cb883751c5ffdbe6d515f3cb4fbe7b7acd", GitTreeState:"clean", BuildDate:"2017-11-08T18:39:33Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"8+", GitVersion:"v1.8.4-gke.1", GitCommit:"04502ae78d522a3d410de3710e1550cfb16dad4a", GitTreeState:"clean", BuildDate:"2017-12-08T17:24:53Z", GoVersion:"go1.8.3b4", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by tamalsaha** [comment]
Fixed in 0.6.1

**Comment by tamalsaha** [comment]
Thanks for the quick error report :)
